src/main/scala/com/campudus/tableaux/helper/ResultChecker.scala

```diff
@@ -4,9 +4,9 @@ import com.campudus.tableaux.{DatabaseException, NotFoundInDatabaseException}
 import org.vertx.scala.core.json.{JsonArray, JsonObject}
 
 /**
- * Checks if database result changed
- * something or not.
- */
+  * Checks if database result changed
```

Comment:
Not sure if I like the Scaladoc formatting... It's the "official" way of doing this, but it also is one character less due to the extra space... :)